--- /tmp/etc.tmp.4//etc/debfoster.conf	1970-01-01 01:00:00.000000000 +0100
+++ /etc/debfoster.conf	2015-03-13 23:48:00.211155858 +0000
@@ -0,0 +1,124 @@
+# InstallCmd = apt-get install
+# Command invoked with a number of packages on the command line.
+# The command is not passed to /bin/sh but invoked like xargs(1)
+# with a number of packages as extra options.
+
+# RemoveCmd = apt-get --purge remove
+# Like InstallCmd but for removing packages.
+
+# InfoCmd = dpkg -s
+# Like InstallCmd but called with a single package as an argument
+# to display information on.
+
+# KeeperFile = /var/lib/debfoster/keepers
+# The file where the list of orphans is stored.  You can use this
+# file for reference when installing a machine or even to make
+# identical Debian installs.
+
+# DpkgStatus = /var/lib/dpkg/status
+# The file where dpkg(8) stores its information about which pack-
+# ages are more or less installed.  This value can usually be left
+# untouched.
+
+# DpkgAvailable = /var/lib/dpkg/available
+# The file where dpkg(8) stores its information about which pack-
+# ages are available.  This value can usually be left untouched.
+
+# MaxPriority = standard
+# Any packages with a priority greater than this value will be con-
+# sidered too basic to ask questions about. With the special value
+# `ANY' you can indicate that all known priorities should be con-
+# sidered too important to ask questions about.
+# Possible values: required, important, standard, optional, extra.
+
+# UseHold = yes
+# Use the hold attribute from the Status: line.  Packages with this
+# attribute won't ever be upgraded by apt, so it's safe to assume
+# that you want to keep it.
+
+# UseEssential = yes
+# Use the Essential: line from dpkg(8)'s  status file.  Most pack-
+# ages which are marked essential shouldn't be removed anyway, so
+# if you don't want to be bothered with it, enable this option.
+
+# UsePreDepends = yes
+# A package that pre-depends on another package requires the latter
+# during installation.  This option will make debfoster count these
+# pre-dependencies as ordinary dependencies.  If you frequently up-
+# date your packages you may want to keep an eye out for pre-de-
+# pended packages that have become obsolete.
+
+UseRecommends = no
+# Recommended packages would be installed together with the package
+# that recommends them in all usual setups.  This option will make
+# debfoster count these recommendations as real dependencies.  En-
+# abling this option will enable you to better manage packages
+# which were installed because another package recommended them.
+
+# UseSuggests = no
+# Packages suggested by another package usually enhance the func-
+# tion of the latter or have a related function which may be useful
+# in combination with the package that suggested them.  This option
+# will make debfoster count these suggestions as real dependencies.
+# Using this option will result in even fewer questions being asked.
+
+# UseTasks = no
+# Make tasks visible as packages. This will make tasks that are
+# selectable using tasksel(1) appear as packages named
+# task-<label>.  debfoster will treat them as if they were normal
+# packages. Tasks cannot be removed but marking a task for removal
+# will stop debfoster asking questions about it.
+
+# KeepSections =
+# NokeepSections =
+# All debian packages are assigned to a section like libs, utils,
+# contrib/games, etc. If you want to have the same default handling
+# for all packages in a particular section, you can add it to either
+# the KeepSections or the NokeepSections list. Note that the first
+# part in a two-part section name, e.g. "contrib/xxx", is not matched.
+#
+# For example, if you want to keep all games and sounds packages by
+# default and delete everything to do with science, you can add the
+# following lines:
+#	KeepSections = games, sounds
+#	NokeepSections = science
+#
+# A perhaps more useful example for cleaning up unused libraries is:
+#	NokeepSections = libs, oldlibs
+#
+# Note that a package can only be deleted if no other package depends
+# on it and no other rule causes it to be kept, regardless of the
+# NokeepSections setting.
+
+# GuessDepends =
+# Larger applications are often separated in to multiple packages
+# and named with names like xyzzy, xyzzy-doc, xyzzy-dev. If you
+# don't want to answer questions about xyzzy-extension, you can add
+# the extension to the GuessDepends list. In this case, debfoster
+# will keep the extension package if the base package is kept.
+# Otherwise it will be deleted (assuming it has no dependents).
+# For example:
+#	GuessDepends = dev, doc
+
+# NegativeKeepers = no
+# Debfoster remembers packages that are explictly added or removed.
+# Sometimes system administrators install certain packages temporarily,
+# and don't want them to be removed automatically each time debfoster
+# is run. Setting this option to `no' makes debfoster ignore negative
+# entries in the keeper file.
+
+# Verbose = no
+# Using this option has the same result as having -v on the command
+# line.  It will make debfoster show which packages have disap-
+# peared or have become a dependency.
+
+# Force = no
+# Corresponds to the -f command line option: preempts questions and
+# answers them with `No'. Additionally, packages that are on the
+# debfoster list but are not currently installed will be installed.
+
+# Quiet = no
+# Having this option (which has the same meaning as the -q command
+# line argument) in your configuration file more or less defeats
+# the purpose of debfoster although the KeeperFile is still kept
+# up-to-date. All packages will be kept without asking questions.
--- /tmp/etc.tmp.4//etc/default/docker	2016-06-01 17:01:51.000000000 +0100
+++ /etc/default/docker	2016-01-15 21:35:33.787464589 +0000
@@ -1,18 +1,15 @@
 # Docker Upstart and SysVinit configuration file
 
-#
-# THIS FILE DOES NOT APPLY TO SYSTEMD
-#
-#   Please see the documentation for "systemd drop-ins":
-#   https://docs.docker.com/engine/articles/systemd/
-#
-
 # Customize location of Docker binary (especially for development testing).
 #DOCKER="/usr/local/bin/docker"
 
 # Use DOCKER_OPTS to modify the daemon startup options.
 #DOCKER_OPTS="--dns 8.8.8.8 --dns 8.8.4.4"
 
+# don't use tiny /var/lib/docker
+# https://github.com/docker/docker/issues/16256
+DOCKER_OPTS="-g /home/docker --exec-opt native.cgroupdriver=cgroupfs"
+
 # If you need Docker to use an HTTP proxy, it can also be specified here.
 #export http_proxy="http://127.0.0.1:3128/"
 
--- /tmp/etc.tmp.4//etc/laptop-mode/conf.d/intel_pstate.conf	2016-03-27 17:18:01.000000000 +0100
+++ /etc/laptop-mode/conf.d/intel_pstate.conf	2015-05-15 21:04:25.328811911 +0100
@@ -37,6 +37,6 @@
 LM_AC_INTEL_PSTATE_NO_TURBO=0 # Disable "Turbo Boost"?
 
 #On battery
-BATT_INTEL_PSTATE_PERF_MIN_PCT=0 # Minimum performance, in percent
+BATT_INTEL_PSTATE_PERF_MIN_PCT=5 # Minimum performance, in percent
 BATT_INTEL_PSTATE_PERF_MAX_PCT=50 # Maximum performance, in percent
 BATT_INTEL_PSTATE_NO_TURBO=1 # Disable "Turbo Boost"?
--- /tmp/etc.tmp.4//etc/lightdm/lightdm.conf	2016-06-28 21:50:09.000000000 +0100
+++ /etc/lightdm/lightdm.conf	2016-04-13 22:08:13.003168499 +0100
@@ -41,7 +41,7 @@
 # [Seat:*] matches all seats and is applied first.
 # [Seat:seat0] matches the seat named "seat0".
 # [Seat:seat-thin-client*] matches all seats that have names that start with "seat-thin-client".
-#
+[SeatDefaults]
 # type = Seat type (xlocal, xremote, unity)
 # pam-service = PAM service to use for login
 # pam-autologin-service = PAM service to use for autologin
@@ -60,13 +60,13 @@
 # unity-compositor-command = Unity compositor command to run (can also contain arguments e.g. unity-system-compositor -special-option)
 # unity-compositor-timeout = Number of seconds to wait for compositor to start
 # greeter-session = Session to load for greeter
-# greeter-hide-users = True to hide the user list
-# greeter-allow-guest = True if the greeter should show a guest login option
-# greeter-show-manual-login = True if the greeter should offer a manual login option
+greeter-hide-users = False # to hide the user list
+greeter-allow-guest = False # if the greeter should show a guest login option
+greeter-show-manual-login = False # if the greeter should offer a manual login option
 # greeter-show-remote-login = True if the greeter should offer a remote login option
 # user-session = Session to load for users
 # allow-user-switching = True if allowed to switch users
-# allow-guest = True if guest login is allowed
+allow-guest = False # if guest login is allowed
 # guest-session = Session to load for guests (overrides user-session)
 # session-wrapper = Wrapper script to run session with
 # greeter-wrapper = Wrapper script to run greeter with
@@ -132,7 +132,6 @@
 # port = UDP/IP port to listen for connections on
 # listen-address = Host/address to listen for XDMCP connections (use all addresses if not present)
 # key = Authentication key to use for XDM-AUTHENTICATION-1 or blank to not use authentication (stored in keys.conf)
-# hostname = Hostname to report to XDMCP clients (defaults to system hostname if unset)
 #
 # The authentication key is a 56 bit DES key specified in hex as 0xnnnnnnnnnnnnnn.  Alternatively
 # it can be a word and the first 7 characters are used as the key.
@@ -142,7 +141,6 @@
 #port=177
 #listen-address=
 #key=
-#hostname=
 
 #
 # VNC Server configuration
--- /tmp/etc.tmp.4//etc/lightdm/lightdm-gtk-greeter.conf	2015-06-21 10:15:27.000000000 +0100
+++ /etc/lightdm/lightdm-gtk-greeter.conf	2015-12-17 23:41:16.663751781 +0000
@@ -45,11 +45,12 @@
 #  transition-duration = overrides default value
 #
 [greeter]
-#background=
+background=/usr/share/images/desktop-base/spacefun-wallpaper-widescreen.svg
+hide-user-image=true
 #user-background=
-#theme-name=
+theme-name=Moblin-Netbook
 #icon-theme-name=
-#font-name=
+font-name="sans-serif 18"
 #xft-antialias=
 #xft-dpi=
 #xft-hintstyle=
@@ -60,3 +61,4 @@
 #reader=
 #position=
 #screensaver-timeout=
+laptop=true
\ No newline at end of file
--- /tmp/etc.tmp.4//etc/modprobe.d/that-fucking-beep.conf	1970-01-01 01:00:00.000000000 +0100
+++ /etc/modprobe.d/that-fucking-beep.conf	2014-07-10 22:57:39.295322281 +0100
@@ -0,0 +1 @@
+blacklist pcspkr
--- /tmp/etc.tmp.4//etc/X11/xorg.conf.d/50-synaptics.conf	1970-01-01 01:00:00.000000000 +0100
+++ /etc/X11/xorg.conf.d/50-synaptics.conf	2015-12-13 22:30:54.933946711 +0000
@@ -0,0 +1,7 @@
+Section "InputClass"
+	Identifier "touchpad"
+	MatchProduct "DLL0665:01 06CB:76AD UNKNOWN"
+	Driver "libinput"
+	Option	"Tapping"	"on"
+	Option	"AccelSpeed"	"1"
+EndSection
--- /tmp/etc.tmp.4//etc/systemd/system/i3lock.service	1970-01-01 01:00:00.000000000 +0100
+++ /etc/systemd/system/i3lock.service	2014-07-10 00:21:46.483729909 +0100
@@ -0,0 +1,14 @@
+#systemctl enable i3lock.service
+
+[Unit]
+Description=i3lock
+Before=sleep.target
+
+[Service]
+User=fommil
+Type=forking
+Environment=DISPLAY=:0
+ExecStart=/usr/bin/i3lock -c 000000
+
+[Install]
+WantedBy=sleep.target
--- /tmp/etc.tmp.4//etc/systemd/system/sleep.target.wants/i3lock.service	1970-01-01 01:00:00.000000000 +0100
+++ /etc/systemd/system/sleep.target.wants/i3lock.service	2014-07-10 00:21:46.483729909 +0100
@@ -0,0 +1,14 @@
+#systemctl enable i3lock.service
+
+[Unit]
+Description=i3lock
+Before=sleep.target
+
+[Service]
+User=fommil
+Type=forking
+Environment=DISPLAY=:0
+ExecStart=/usr/bin/i3lock -c 000000
+
+[Install]
+WantedBy=sleep.target
--- /tmp/etc.tmp.4//etc/default/keyboard	1970-01-01 01:00:00.000000000 +0100
+++ /etc/default/keyboard	2016-06-11 16:05:59.793138329 +0100
@@ -0,0 +1,10 @@
+# KEYBOARD CONFIGURATION FILE
+
+# Consult the keyboard(5) manual page.
+
+XKBMODEL="pc101"
+XKBLAYOUT="us"
+XKBVARIANT="dvorak-alt-intl"
+XKBOPTIONS=""
+
+BACKSPACE="guess"
--- /tmp/etc.tmp.4//etc/iptables/rules.v4	1970-01-01 01:00:00.000000000 +0100
+++ /etc/iptables/rules.v4	2015-04-03 20:49:13.208315272 +0100
@@ -0,0 +1,30 @@
+# Generated by iptables-save v1.4.21 on Wed Mar 25 22:12:18 2015
+*nat
+:PREROUTING ACCEPT [1040850:81532528]
+:INPUT ACCEPT [33451:1921332]
+:OUTPUT ACCEPT [48606:3503074]
+:POSTROUTING ACCEPT [48606:3503074]
+:DOCKER - [0:0]
+-A PREROUTING -m addrtype --dst-type LOCAL -j DOCKER
+-A OUTPUT ! -d 127.0.0.0/8 -m addrtype --dst-type LOCAL -j DOCKER
+-A POSTROUTING -s 172.17.0.0/16 ! -o docker0 -j MASQUERADE
+COMMIT
+# Completed on Wed Mar 25 22:12:18 2015
+# Generated by iptables-save v1.4.21 on Wed Mar 25 22:12:18 2015
+*filter
+:INPUT ACCEPT [0:0]
+:FORWARD ACCEPT [0:0]
+:OUTPUT ACCEPT [0:0]
+-A INPUT -i lo -j ACCEPT
+-A INPUT -d 127.0.0.0/8 ! -i lo -j REJECT --reject-with icmp-port-unreachable
+-A INPUT -m state --state RELATED,ESTABLISHED -j ACCEPT
+-A INPUT -p tcp -m tcp --dport 80 -j ACCEPT
+-A INPUT -p tcp -m tcp --dport 443 -j ACCEPT
+-A INPUT -p tcp -m state --state NEW -m tcp --dport 22 -j ACCEPT
+-A INPUT -p icmp -m icmp --icmp-type 8 -j ACCEPT
+-A INPUT -m limit --limit 5/min -j LOG --log-prefix "iptables denied: " --log-level 7
+-A INPUT -j REJECT --reject-with icmp-port-unreachable
+-A FORWARD -j REJECT --reject-with icmp-port-unreachable
+-A OUTPUT -j ACCEPT
+COMMIT
+# Completed on Wed Mar 25 22:12:18 2015
--- /tmp/etc.tmp.4//etc/apt/sources.list.d/azul.list	1970-01-01 01:00:00.000000000 +0100
+++ /etc/apt/sources.list.d/azul.list	2016-04-17 20:43:05.770935631 +0100
@@ -0,0 +1 @@
+deb http://repos.azulsystems.com/ubuntu stable main
--- /tmp/etc.tmp.4//etc/apt/sources.list.d/docker.list	1970-01-01 01:00:00.000000000 +0100
+++ /etc/apt/sources.list.d/docker.list	2016-01-06 21:19:32.813830985 +0000
@@ -0,0 +1 @@
+deb https://apt.dockerproject.org/repo debian-stretch main
--- /tmp/etc.tmp.4//etc/ld.so.conf.d/mkl.conf	1970-01-01 01:00:00.000000000 +0100
+++ /etc/ld.so.conf.d/mkl.conf	2016-04-24 12:26:51.556649331 +0100
@@ -0,0 +1,4 @@
+/opt/intel/mkl/lib/intel64/
+/opt/intel/ism/qt/intel64/plugins/platforms/
+/opt/intel/lib/intel64
+/opt/intel/mkl/lib/intel64
--- /tmp/etc.tmp.4//etc/modprobe.d/xps13-headphones.conf	1970-01-01 01:00:00.000000000 +0100
+++ /etc/modprobe.d/xps13-headphones.conf	2014-12-29 12:37:35.894622389 +0000
@@ -0,0 +1,3 @@
+#http://voices.canonical.com/david.henningsson/2014/03/07/headset-jacks-on-newer-laptops/
+# also see https://askubuntu.com/questions/132440
+options snd-hda-intel model=dell-headset-multi
--- /tmp/etc.tmp.4//etc/modprobe.d/xps13-touchpad.conf	1970-01-01 01:00:00.000000000 +0100
+++ /etc/modprobe.d/xps13-touchpad.conf	2015-12-13 22:29:59.173794843 +0000
@@ -0,0 +1 @@
+blacklist psmouse
\ No newline at end of file
--- /tmp/etc.tmp.4//etc/sudoers.d/apt	1970-01-01 01:00:00.000000000 +0100
+++ /etc/sudoers.d/apt	2015-11-21 16:09:54.086582504 +0000
@@ -0,0 +1,3 @@
+fommil  ALL = NOPASSWD: /usr/bin/apt-get
+fommil  ALL = NOPASSWD: /usr/bin/aptitude
+fommil  ALL = NOPASSWD: /usr/bin/debfoster
--- /tmp/etc.tmp.4//etc/sudoers.d/power	1970-01-01 01:00:00.000000000 +0100
+++ /etc/sudoers.d/power	2015-03-28 13:57:01.515459755 +0000
@@ -0,0 +1,3 @@
+fommil  ALL = NOPASSWD: /usr/sbin/poweroff
+fommil  ALL = NOPASSWD: /usr/sbin/reboot
+fommil  ALL = NOPASSWD: /usr/sbin/systemctl
--- /tmp/etc.tmp.4//etc/X11/xorg.conf.d/50-synaptics.conf	1970-01-01 01:00:00.000000000 +0100
+++ /etc/X11/xorg.conf.d/50-synaptics.conf	2015-12-13 22:30:54.933946711 +0000
@@ -0,0 +1,7 @@
+Section "InputClass"
+	Identifier "touchpad"
+	MatchProduct "DLL0665:01 06CB:76AD UNKNOWN"
+	Driver "libinput"
+	Option	"Tapping"	"on"
+	Option	"AccelSpeed"	"1"
+EndSection
